test/Microsoft.DotNet.Watcher.Tools.FunctionalTests/NoDepsAppTests.cs

```diff
@@ -51,7 +52,7 @@ public void RestartProcessOnFileChange()
             }
         }
 
-        [Fact(Skip = "aspnet/dotnet-watch#98")]
+        [Fact]
```

Comment:
Suggest doing this in a separate commit. That way, `git log` can show dc9fead has been reverted.